gcc/testsuite/jit.dg/test-reflection.c

```diff
@@ -60,6 +105,7 @@ verify_code (gcc_jit_context *ctxt, gcc_jit_result *result)
   CHECK_VALUE (gcc_jit_vector_type_get_element_type(vector_type), double_type);
   CHECK_VALUE (gcc_jit_vector_type_get_num_units(vector_type), 4);
   CHECK (!gcc_jit_type_is_integral(vec_type));
+  CHECK (gcc_jit_type_is_float(vec_type));
```

Comment:
I'm not sure this makes sense that this returns true on a vector type.
Is this the same for `gcc_jit_type_is_integral`?

Reply:
`vector_type` inherits from `decorated_type`, whose `is_float` method calls the `is_float()` method of the vector element.
I will add additional check for vectors.